bitsandbytes/diagnostics/cuda.py

```diff
@@ -105,37 +108,58 @@ def find_cudart_libraries() -> Iterator[Path]:
 
 
 def print_cuda_diagnostics(cuda_specs: CUDASpecs) -> None:
-    print(
-        f"PyTorch settings found: CUDA_VERSION={cuda_specs.cuda_version_string}, "
-        f"Highest Compute Capability: {cuda_specs.highest_compute_capability}.",
-    )
+    if not HIP_ENVIRONMENT:
+        print(
+            f"PyTorch settings found: CUDA_VERSION={cuda_specs.cuda_version_string}, "
+            f"Highest Compute Capability: {cuda_specs.highest_compute_capability}.",
+        )
+    else:
+        print(f"PyTorch settings found: ROCM_VERSION={cuda_specs.cuda_version_string}")
```

Comment:
Instead of these printing functions needing to know whether HIP or CUDA is in play, I think `CUDASpecs` should have a new field for that (e.g. `platform: str` that could be `cuda` or `hip`) for now.

What's the reason for not printing the highest compute capability? Is that not available for HIP?

Reply:
I have created separate functions based on the next comment, so the condition checks won't be needed anymore.

Highest compute capability is cuda specific, its not applicable to us